Utils/HookBase.cs

```diff
@@ -0,0 +1,32 @@
+﻿using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SpecFlowProject_MarsProject.Utils
+{
+    public class HookBase
+    {
+        protected static IWebDriver driver;
+        
+        [BeforeScenario]
+        public void BeforeScenario()
+        {
+            // Initialize WebDriver (Chrome in this case)
+            driver = new ChromeDriver();
```

Comment:
ideally you should create a Common driver object here and then move this logic to the common driver